grass7/gui/wxpython/wx.metadata/g.gui.cswbrowser/g.gui.cswbrowser.py

```diff
@@ -9,17 +9,18 @@
 
 @author Matej Krejci <matejkrejci gmail.com> (GSoC 2015)
 """
+
 import grass.script as grass
+from grass.script.setup import set_gui_path
+
+grass.utils.set_path(modulename='wx.metadata', dirname='mdlib', path='..')
 
-from grass.pygrass.utils import set_path
-set_path(modulename='wx.metadata', dirname='mdlib')
+from mdlib.cswlib import CSWBrowserPanel, CSWConnectionPanel
 
 import wx
```

Comment:
```suggestion
import wx

#%module
#% description: Graphical CSW metadata browser.
#% keyword: general
#% keyword: GUI
#% keyword: metadata
#%end
```

Silence: "WARNING: Bug in UI description. Missing module description"

--> might be extended

Reply:
I fixed it.